server/protocols/webadmin/cli/james_cli

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python3
+
+import click
+import json
+import requests as req
+import sys
+
+
+def print_json(obj):
+    parsed = json.loads(obj.text)
+    print(json.dumps(parsed, indent=4, sort_keys=True))
+
+
+@click.group()
+@click.option("--ip", default="127.0.0.1", show_default=True, help="Set ip address")
+@click.option("--port", default="8000", show_default=True, help="Set port")
```

Comment:
Is it possible with click to specify a short form of `--port` being `-p`, or `--host` being `-h` ?